azure/datalake/store/transfer.py

```diff
@@ -387,7 +387,8 @@ def _update(self, future):
                     merge_future = self._submit(
                         self._merge, self._adlfs, dst,
                         [chunk for chunk, _ in sorted(cstates.objects,
-                                                      key=lambda obj: obj[1])])
+                                                      key=lambda obj: obj[1])], 
+                        overwrite=self._parent._overwrite)
```

Comment:
Two private attributes [chained ](https://en.wikipedia.org/wiki/Law_of_Demeter) makes me nervous. But yes, it's not a proof but just a warning :)